src/main/scala/akka/persistence/jdbc/query/scaladsl/JdbcReadJournal.scala

```diff
@@ -142,8 +142,8 @@ class JdbcReadJournal(config: Config, configPath: String)(implicit val system: E
         case _ =>
           delaySource
             .flatMapConcat { _ =>
-              currentJournalEventsByPersistenceId(persistenceId, from, toSequenceNr)
-                .take(readJournalConfig.maxBufferSize)
+              val to = Math.min(from + (readJournalConfig.maxBufferSize - 1), toSequenceNr)
```

Comment:
Since it is possible to physically delete events. It could be possible that a persistent actor has its first event with the lowest sequence number 101 (in this case the first 100 events have been deleted).

If we compute the to value this way, and we want to query all events (from 1 to Long.MaxValue), then our first batch would be empty.
It seems that the stream would still continue, so eventually the remaining data would be included int the results.
However since the `delaySource` will keep on triggering delays, each batch will have a delay between them.

Reply:
Yes indeed. Do you think we should optimize it and first fetch the lowest non-deleted event seqNum for the passed persistence id? 

On the other hand, someone using this method is really controlling the environment by explicitly passing the `from` and `to` sequence numbers. So, I would not expect people calling it with a `fromSeqNr` for a delete event.

Reply:
quoting myself...
> So, I would not expect people calling it with a fromSeqNr for a delete event.

And that's when people surprises you. We never know how it will be used.

Reply:
@richbirch / @WellingR, I'm ok with the currenct implementation. Maybe we should add in the scaladoc that query will be delayed if you have deleted events and you querying using a `fromSequeceNr` from a delete event.

Reply:
And also a notice on the README section covering `eventsByPersistenceId`
https://github.com/dnvriend/akka-persistence-jdbc#eventsbypersistenceidquery-and-currenteventsbypersistenceidquery